data/102/2613/en_US.json

```diff
@@ -5,7 +5,7 @@
 			"message": "The information entered is not correct.\nYou may have entered the ID of another\nservice.\n\nPlease enter the details again, making\nsure that the Nintendo Network ID,\npassword, and e-mail address are\ncorrect.",
 			"short_description": "Invalid account information provided. Either your username, password, or email address does not match",
 			"long_description": "Unknown cause.",
-			"short_solution": "Enter corrected information and try again",
+			"short_solution": "Enter corrected information and try again. You might also not be connected to Pretendo Network.",
```

Comment:
I would change this to be a bit more generic. If someone is on NN and tries to use their PNID, or connected to Pretendo and tries to use their NNID, they get the same error here. But this makes it sound like it only happens when not using Pretendo